prow/config/config.go

```diff
@@ -1573,6 +1574,64 @@ func (cfg *SlackReporter) DefaultAndValidate() error {
 	return nil
 }
 
+// RocketChatReporter represents the config for the RocketChat reporter. The channel can be overridden
+// on the job via the .reporter_config.rocketchat.channel property.
+type RocketChatReporter struct {
+	JobTypesToReport                 []prowapi.ProwJobType `json:"job_types_to_report,omitempty"`
+	prowapi.RocketChatReporterConfig `json:",inline"`
+}
+
+// RocketChatReporterConfigs represents the config for the RocketChat reporter(s).
+// Use `org/repo`, `org` or `*` as key and an `RocketChatReporter` struct as value.
+type RocketChatReporterConfigs map[string]RocketChatReporter
+
+// RocketChatReporterConfig represents the config for the RocketChat reporter(s).
+// Use `org/repo`, `org` or `*` as key and an `RocketChatReporter` struct as value.
+type RocketChatReporterConfig map[string]RocketChatReporter
```

Comment:
Why do we need both? Could you add the reasoning in the comment for one of them? (Or better, just delete one of them?)

Reply:
I removed it.